web/src/app/app.module.ts

```diff
@@ -32,6 +32,7 @@ import { SocialCardsComponent } from './components/social-cards/social-cards.com
     HomeComponent,
     NavbarComponent,
     SocialCardsComponent,
+    HttpClientModule
```

Comment:
`declarations` I think is incorrect and it should be under `imports`. See our other discussion https://github.com/DashboardHub/TribeDashboard/pull/76#discussion_r283222655

Reply:
Apologies @eddiejaoude. Changed it.